.eslintrc.js

```diff
@@ -17,35 +20,37 @@ module.exports = {
         "CallExpression": { "arguments": "first" },
         "FunctionDeclaration": { "parameters": "first" },
         "FunctionExpression": { "parameters": "first" },
-      }
+      },
     ],
     "lodash/prefer-lodash-method": [0],
     "lodash/prefer-noop": [0],
     "max-len": ["error", { "code": 110 }],
     "operator-linebreak": ["error", "before"],
     // override airbnb-base dev dependencies, latest version does not white list __mocks__
-    'import/no-extraneous-dependencies': ['error', {
-      devDependencies: [
-        'test/**', // tape, common npm pattern
-        'tests/**', // also common npm pattern
-        'spec/**', // mocha, rspec-like pattern
-        '**/__tests__/**', // jest pattern
-        '**/__mocks__/**', // jest pattern
-        'test.{js,jsx}', // repos with a single test file
-        'test-*.{js,jsx}', // repos with multiple top-level test files
-        '**/*.{test,spec}.{js,jsx}', // tests where the extension denotes that it is a test
-        '**/jest.config.js', // jest config
-        '**/webpack.config.js', // webpack config
-        '**/webpack.config.*.js', // webpack config
-        '**/rollup.config.js', // rollup config
-        '**/rollup.config.*.js', // rollup config
-        '**/gulpfile.js', // gulp config
-        '**/gulpfile.*.js', // gulp config
-        '**/Gruntfile{,.js}', // grunt config
-        '**/protractor.conf.js', // protractor config
-        '**/protractor.conf.*.js', // protractor config
-      ],
-      optionalDependencies: false,
-    }],
-  }
+    "import/no-extraneous-dependencies": [
+      "error", {
+        "devDependencies": [
+          "test/**", // tape, common npm pattern
+          "tests/**", // also common npm pattern
+          "spec/**", // mocha, rspec-like pattern
+          "**/__tests__/**", // jest pattern
+          "**/__mocks__/**", // jest pattern
+          "test.{js,jsx}", // repos with a single test file
+          "test-*.{js,jsx}", // repos with multiple top-level test files
+          "**/*.{test,spec}.{js,jsx}", // tests where the extension denotes that it is a test
+          "**/jest.config.js", // jest config
+          "**/webpack.config.js", // webpack config
+          "**/webpack.config.*.js", // webpack config
+          "**/rollup.config.*.js", // rollup config
+          "**/rollup.config.*.js", // rollup config
+          "**/gulpfile.js", // gulp config
+          "**/gulpfile.*.js", // gulp config
+          "**/Gruntfile{,.js}", // grunt config
+          "**/protractor.conf.js", // protractor config
+          "**/protractor.conf.*.js", // protractor config
```

Comment:
@nicholaschuayunzhi Did you get these from somewhere, or can I sort alphabetically?

Reply:
I got it off the airbnb config last time.
https://github.com/airbnb/javascript/blob/ae82d7678303bd08242299a4a9e2c47717bf9c6b/packages/eslint-config-airbnb-base/rules/imports.js#L71-L91

It was to allow mock folders to import dev dependencies. At the time of the commit, they had only just committed the change and was not released yet.

Reply:
Looks like it's still not released yet. I'll keep the order then. Thanks!